src/stories/storyContainer.tsx

```diff
@@ -1,10 +1,16 @@
 import React from "react"
 
+interface StoryContainerArgs {
```

Comment:
for the sake of consistency - we usually name the interface of a component `Props` by default
```suggestion
interface Props {
```